client/src/components/breadcrumb/component.tsx

```diff
@@ -15,7 +17,12 @@ const Breadcrumb: React.FC<BreadcrumbProps> = ({ pages }: BreadcrumbProps) => {
   const { mode } = useAppSelector(scenarios);
   const middlePages = [...pages].slice(1, pages.length);
 
-  const handleClick = useCallback(() => dispatch(setMode(pages[0].mode)), [dispatch, pages]);
+  const handleClick = useCallback(() => {
+    if (pages[0].name === 'Analysis') {
```

Comment:
Do not use `name`. The name is more likely to change than the `pathname` or similar.